vita3k/gui/src/controllers_dialog.cpp

```diff
@@ -214,10 +243,17 @@ void draw_controllers_dialog(GuiState &gui, EmuEnvState &emuenv) {
             ImGui::Spacing();
             ImGui::TableSetColumnIndex(2);
             ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%s", "Motion Support");
+            const char *port_names[] = { "1", "2", "3", "4" };
+            static int selected_port = -1;
             for (auto i = 0; i < ctrl.controllers_num; i++) {
                 ImGui::TableNextRow();
                 ImGui::TableSetColumnIndex(0);
-                ImGui::Text("%-8d", i);
+                selected_port = i;
```

Comment:
```suggestion
                int selected_port = i;
```
Last small thing, no need to make a static variable if you always reset it.

Reply:
done